core/src/main/resources/db/changelog/changesets/2024_03_18_localized_email_templates.yaml

```diff
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: "localized_email_templates"
+      author: mbemis
+      changes:
+        - createTable:
+            tableName: localized_email_template
+            columns:
+              - column: { name: id, type: uuid, defaultValueComputed: gen_random_uuid(), constraints: { nullable: false, primaryKey: true } }
+              - column: { name: email_template_id, type: uuid, constraints: { nullable: false } }
+              - column: { name: language, type: text, constraints: { nullable: false } }
+              - column: { name: subject, type: text, constraints: { nullable: false } }
+              - column: { name: body, type: text, constraints: { nullable: false } }
+              - column: { name: created_at, type: timestamp, constraints: { nullable: false } }
+              - column: { name: last_updated_at, type: timestamp, constraints: { nullable: false } }
+        - sql:
+            sql:
+              INSERT INTO localized_email_template (email_template_id, language, subject, body, created_at, last_updated_at)
+              SELECT id, 'en', subject, body, created_at, last_updated_at
+              FROM email_template;
+        - dropColumn:
+            tableName: email_template
+            columnName: subject
+        - dropColumn:
+            tableName: email_template
+            columnName: body
+        - addColumn:
+            tableName: email_template
+            columns:
```

Comment:
outside the scope of this PR, but I suspect "defaultLanguage" should be a property of the PortalEnvironment, not the individual siteContent and emailTemplates

Reply:
Good point, [there's an existing ticket](https://broadworkbench.atlassian.net/browse/JN-863) for adding defaultLanguages to PortalEnvironments- I think I'll remove the defaultLanguage stuff from this PR since it doesn't really do anything yet and will make the follow-up a bit easier